utils/utils.go

```diff
@@ -0,0 +1,23 @@
+package utils
```

Comment:
Rename package to `fileutils`? It's better to have multiple focused utility / common packages, rather than a single utils with many unrelated functions.

Reply:
that's fair, I couldn't think of a good name.